phpcs.xml

```diff
@@ -21,27 +21,6 @@
 		<exclude name="PHPCompatibility.PHP.NewKeywords.t_namespaceFound" />
 	</rule>
 
-	<rule ref="WordPress">
```

Comment:
I'm missing why this rule was removed. Is it not adding anything? I tried re-enabling it locally to test it and doesn't seem to make any difference, but wanted to ask to be sure.

Reply:
That was unintentional … I'll put it back. I was hacking on this xml file heavily trying to confirm the right rulesets were applying, thanks for catching.